packages/react/src/components/Dialog/Dialog.tsx

```diff
@@ -142,7 +144,21 @@ class Dialog extends AutoControlledComponent<WithAsProp<DialogProps>, DialogStat
   triggerRef = React.createRef<HTMLElement>()
 
   getInitialAutoControlledState(): DialogState {
-    return { open: false }
+    return {
+      contentId: _.uniqueId('dialog-content-'),
```

Comment:
Can we store this strings to some statics, or reuse the classnames if they exists?

Reply:
I don't there a big benefit because we don't want to expose them to clients